wrapanapi/systems/rhevm.py

```diff
@@ -451,10 +453,12 @@ def add_disk(self, storage_domain=None, size=None, interface='VIRTIO', format=No
         """
         disk_attachments_service = self.api.disk_attachments_service()
         disk_attach = types.DiskAttachment(
-            disk=types.Disk(format=types.DiskFormat(format),
+            disk=types.Disk(name=str(name),
```

Comment:
If `name=None` by default we will get disk named `None` in here. Should we prevent that?

Reply:
nope, ovirt will pick a name - `None` is the default kwarg for the type in ovirt.

Reply:
OH I see what you mean, if we're directly casting string. Yeah I'll remove that.